active-fedora.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
   s.add_dependency "active-triples", '~> 0.11.0'
   s.add_dependency "deprecation"
   s.add_dependency "ldp", '~> 0.6.0'
+  s.add_dependency "ruby-progressbar", '~> 1.0'
```

Comment:
@jrochkind I like the reasoning, and can get behind its inclusion.

However, I still want to see a spec on its usage. If we are adding a dependency that ripples upstream, I want to make sure we have at least one test against that dependency.